ci/releases.yml

```diff
@@ -61,3 +61,18 @@ release commit messages:
       - /^release\//
   script:
     - ci/scripts/check_release_commit_messages.sh
+
+# Messaging system deploy to production
+msg-system codesign deploy:
+  stage: deploy to production
+  when: manual
+  needs:
+    - "msg-system config sign stable"
+  only:
+    refs:
+      - codesign
+  tags:
+    - darwin
+  script:
+    - aws s3 cp s3://data.trezor.io/config/stable/config.v1.jws s3://data.trezor.io/config/stable/config.v1.jws.old
```

Comment:
Just an idea. Some sort of "backup". Since we bump the sequence number this is not meant as something we can return it rather than if we the new config was incorrectly signed therefore not parsed. Maybe not worth it?

Reply:
I am not sure if I understand the question correctly. However is there a possibility that the config file is signed incorrectly?

Reply:
Not really.. And we have lints that make sure the config is valid json and valid json schema right? Just just one case I can image is that we incorrectly set the sequence to e.g. 0, sign and deploy. This way we could easily roll back. But since Suite just ignores the invalid configs, I agree it is too much for such use case. I'll remove it..

Reply:
6249dfc30d512a151cf788e52ed371fca3bf4aa6

Reply:
agree that we don't need a copy of the old one as we should always sign it correctly.